Changes.md

```diff
@@ -27,7 +27,7 @@ Improvements
   - Renamed and reordered parameters and created sections to match Blender.
   - Changed all the Cycles items in the Gaffer tab menu to title case.
 - PythonCommand : Added standard popup menu to `command` widget in NodeEditor.
-- LightEditor : Added the ability to register columns for editing any shader in the light's shader network with an Edit Scope or a `ShaderTweaks` node. The parameter to edit is registered in the form `shaderName.parameterName` when calling `GafferSceneUI.LightEditor.registerParameter`.
+- LightEditor : Added the ability to register columns for editing any shader in the light's shader network. The parameter to edit is registered in the form `shaderName.parameterName` when calling `GafferSceneUI.LightEditor.registerParameter`.
```

Comment:
I think the fact that every commit has needed to edit this line suggests that there might be a more logical commit order now we know where we're heading :

1. Add `ShaderPlug::parameterSource()`.
2. Use `parameterSource()` in ParameterInspector.
3. Expose feature in LightEditor.